pkgs/tools/misc/pandoc-drawio-filter/default.nix

```diff
@@ -0,0 +1,32 @@
+{ buildPythonApplication
+, fetchFromGitHub
+, drawio
+, pandocfilters
+, xvfb-run
+, lib
+}:
+
+buildPythonApplication rec {
+  name = "pandoc-drawio-filter";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "tfc";
+    repo = "pandoc-drawio-filter";
+    rev = "55192da626c639874153e643c1e003425083c4a1";
```

Comment:
Why are we not using the tag?

Reply:
There are no tags. Shall i create a tag in the original repo, or are we ok without?